### ISSUE TITLE
Organiser emails being treated as case sensitive

### ISSUE THREAD

**Issue by GirlGeekUpNorth** [DjangoGirls/djangogirls#461]
When organisers apply to host a workshop their emails seem to be being treated as case sensitive. This means that if we have a previous organiser using test@gmail.com and they apply to host another workshop using Test@gmail.com, then they have a second organiser account created for the new workshop.

**Comment by seedhisadak** [comment]
https://github.com/DjangoGirls/djangogirls/pull/513

**Comment by das-g** [comment]
Domain names (`gmail.com` vs. `GMail.com`, `some.example.com` vs. `sOmE.example.com`) should never be treated case sensitive. Whether the user part of an email address (the part in front of the `@`) is case sensitive actually depends on the email provider. Some might have separate mailboxes for `test@example.com` vs. `Test@example.com` that might even belong to different users, while other providers might treat them as addresses for the same mailbox.

**Comment by das-g** [comment]
Also note that when your GMail account is `foobar@gmail.com`, all addresses of the form `foobar+anything@gmail.com` will also point to your GMail inbox. This is however a GMail convention and other email providers might or might not have the same addressing scheme or may employ different variants of it.

Though usually, one can assume that if someone give an address with a different part after the `+` they _don't want_ to be recognized as the same identity as when they gave the address without that or with something different behind the `+`.

**Comment by ramonsaraiva** [comment]
@das-g could you provide an example of an e-mail provider that differentiates test@* from Test@* and points to different inboxes/users?

**Comment by seedhisadak** [comment]
`abc@gmail.com` and `abc+test@gmail.com` can be treated as separate registrations. This gives everyone the ability to create separate registrations with a single email id (at least that is what we have at our firm, where it becomes easier to create test accounts using the same email id).

About lowercase vs uppercase, I am not sure I have ever come across any provider which does this. Please do share some examples.

**Comment by das-g** [comment]
I don't know of any specific current examples and they're probably rare enough that we can somewhat safely ignore this quirk. But it's something we should keep in mind in case it pops up. (The key point is that any provider is allowed to do that, so any day one could pop up that does it. Probably not one offering email accounts to the wider public, but who knows what all companies that provide their employees with email addresses will do.)

**Comment by ramonsaraiva** [comment]
Yep. At the same time I agree it is something viable I doubt providers would allow something awful like that. But you do have a point :-)

**Comment by ramonsaraiva** [comment]
@seedhisadak I also agree that `abc@gmail.com` and `abc+test@gmail.com` should be treated as different users/registrations.
